strings/group_anagram.py

```diff
@@ -0,0 +1,42 @@
+import doctest
+
+def find_anagrams(_list: list):
+    emptyDict = dict()
+    emptylist = []
```

Comment:
```suggestion
    empty_dict = {}  # Python names variables using snake_case.
    empty_list = []
```
But try to come up with better names because these variables have the most value when they are NOT empty.